frontend/src/components/ Footer/styles.module.scss

```
@@ -0,0 +1,91 @@
+@import url('https://fonts.googleapis.com/css?family=Oswald|Playfair+Display|Roboto+Slab&display=swap');
+
+
+footer{
```

Faltó un espacio entre footer y {
```suggestion
footer {
```